httptransport/server.go

```diff
@@ -300,7 +336,7 @@ func unmodified(r *http.Request, v string) bool {
 	return false
 }
 
-// WriterError is a helper that closes over an error that may be returned after
+// writerError is a helper that closes over an error that may be returned after
```

Comment:
Why un-capitalize the start of a sentence?

Reply:
the comment is referring to a private function, caps have meaning.